WooCommerce/Classes/ViewRelated/Products/Edit Product/BottomSheetListSelector/ProductFormBottomSheetActionsFactory.swift

```diff
@@ -0,0 +1,17 @@
+import Yosemite
+
+struct ProductFormBottomSheetActionsFactory {
+    /// Retruns an array of actions that are visible in the product form bottom sheet.
+    static func actions(product: Product, isEditProductsRelease2Enabled: Bool, isEditProductsRelease3Enabled: Bool) -> [ProductFormBottomSheetAction] {
```

Comment:
Not really significant for this but it'd be nice if we avoid `static` methods as much as possible. For `Factory` objects for example, they can be injected for mocking and non-static methods would be better for that use case. 

```swift
let factory = ProductFormBottomSheetActionsFactory(product: product, ...) 
let editForm = ProductFormVC(bottomSheetActionsFactory: factory)
```

Reply:
I usually prefer not to use `static` methods either, but wasn't exactly sure what our conventions are since we also have quite some `static` methods in the codebase 🤔  On the other hand, if we are good with the plan in https://github.com/woocommerce/woocommerce-ios/issues/2317 to have the factory create actions for both the bottom sheet and product form, it'd be nice to pass all the dependencies to the initializer and we can just get the bottom sheet actions (and product form actions in the future) directly. I made an update in https://github.com/woocommerce/woocommerce-ios/pull/2303/commits/353fd12abd6210d830f59252ee8d01668f9f1b78